### DISCUSSION TITLE
键值Logo无法加载本地图片

### DISCUSSION THREAD

**Discussion by AISophon** [Meloong-Git/PCL#2078] [提问] [answered]
![屏幕截图 2023-06-20 134722](https://github.com/Hex-Dragon/PCL2/assets/130284780/683e8084-c105-428a-a053-2fbc6cca9c0c)
如图所示，我尝试使用本地图片加载`自定义帮助文件`中的json文件中的`键值Logo`失败了

但是它似乎只能加载网页图片，如：`https://www.bilibili.com/favicon.ico`，且我的主页文件可以加载本地图片

我不知道怎么办，求大佬指教

**Comment by AISophon** [comment]
我删除`键值Logo`倒是可以正常使用的

**Comment by WForst-Breeze** [comment, marked as answer]
请将相应部分替换为如下内容，即将 “\” 替换为 “/”：
```json
"Logo": "C:/Users/Kdc_P/Documents/Program/PCL/PCL/logo/home.png",
```

**Comment by so1ve** [comment]
如 @WForst-Breeze 所说，你需要改成/来避免转义，或者你可以改成双反斜杠 `\\` 。你现在的json其实是把后一个字符转义了，等于没有分隔符

**Comment by AISophon** [comment]
谢谢你们！
